fix: Plugin Error: java.lang.NoClassDefFoundError: org/apache/commons/text/StringSubstitutor

**Description**

I get the following error when trying to create a new bloc using the plugin.

I have the Bloc plugin version `4.1.1` installed on IntelliJ IDEA Community `2025.1.1`.

**Steps To Reproduce**

1. Go to any folder in a flutter project
2. Right-Click on the folder
3. Scroll to New -> Bloc Class. Type the name and set the Style to Equatable
4. Click 'Ok'
5. IntelliJ shows an alert that an error occurred with the error message below.

**Expected Behavior**
A new Bloc should be created with the given name.

**Screenshots**

IntelliJ Version:
<img width="492" alt="Image" src="https://github.com/user-attachments/assets/e070a58b-4d8c-4b47-9957-9195d7cb78d7" />

Plugin Version:
<img width="747" alt="Image" src="https://github.com/user-attachments/assets/72d582b0-d425-492d-aef5-f78b67acb432" />

**Additional Context**
None

**Error Message**
```
java.lang.NoClassDefFoundError: org/apache/commons/text/StringSubstitutor
	at com.bloc.intellij_generator_plugin.generator.BlocGenerator.generate(BlocGenerator.kt:44)
	at com.bloc.intellij_generator_plugin.action.GenerateBlocAction.createSourceFile(GenerateBlocAction.kt:62)
	at com.bloc.intellij_generator_plugin.action.GenerateBlocAction.generate$lambda$4$lambda$3(GenerateBlocAction.kt:47)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.lambda$executeCommand$2(CoreCommandProcessor.java:228)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction$lambda$6(AnyThreadWriteThreadingSupport.kt:274)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWithTemporaryThreadLocal(AnyThreadWriteThreadingSupport.kt:204)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:274)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:222)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteIntentReadAction(ApplicationImpl.java:1009)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:224)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:179)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:169)
	at com.intellij.openapi.command.impl.CoreCommandProcessor.executeCommand(CoreCommandProcessor.java:155)
	at com.bloc.intellij_generator_plugin.action.GenerateBlocAction.generate$lambda$4(GenerateBlocAction.kt:45)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction$lambda$13(AnyThreadWriteThreadingSupport.kt:613)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction$lambda$15(AnyThreadWriteThreadingSupport.kt:623)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWithTemporaryThreadLocal(AnyThreadWriteThreadingSupport.kt:204)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:623)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteAction(AnyThreadWriteThreadingSupport.kt:613)
	at com.intellij.openapi.application.impl.ApplicationImpl.runWriteAction(ApplicationImpl.java:988)
	at com.bloc.intellij_generator_plugin.action.GenerateBlocAction.generate(GenerateBlocAction.kt:44)
	at com.bloc.intellij_generator_plugin.action.GenerateBlocAction.onGenerateBlocClicked(GenerateBlocAction.kt:28)
	at com.bloc.intellij_generator_plugin.action.GenerateBlocDialog.doOKAction(GenerateBlocDialog.java:40)
	at com.intellij.openapi.ui.DialogWrapper$OkAction.doAction(DialogWrapper.java:2011)
	at com.intellij.openapi.ui.DialogWrapper$DialogWrapperAction.actionPerformed(DialogWrapper.java:1962)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at java.desktop/javax.swing.AbstractButton$Handler.actionPerformed(AbstractButton.java:2314)
	at java.desktop/javax.swing.DefaultButtonModel.fireActionPerformed(DefaultButtonModel.java:407)
	at java.desktop/javax.swing.DefaultButtonModel.setPressed(DefaultButtonModel.java:262)
	at java.desktop/javax.swing.AbstractButton.doClick(AbstractButton.java:374)
	at java.desktop/javax.swing.plaf.basic.BasicRootPaneUI$Actions.actionPerformed(BasicRootPaneUI.java:259)
	at java.desktop/javax.swing.SwingUtilities.notifyAction(SwingUtilities.java:1810)
	at java.desktop/javax.swing.JComponent.processKeyBinding(JComponent.java:2952)
	at java.desktop/javax.swing.KeyboardManager.fireBinding(KeyboardManager.java:309)
	at java.desktop/javax.swing.KeyboardManager.fireKeyboardAction(KeyboardManager.java:251)
	at java.desktop/javax.swing.JComponent.processKeyBindingsForAllComponents(JComponent.java:3045)
	at java.desktop/javax.swing.JComponent.processKeyBindings(JComponent.java:3037)
	at java.desktop/javax.swing.JComponent.processKeyEvent(JComponent.java:2914)
	at java.desktop/java.awt.Component.processEvent(Component.java:6450)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5043)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4871)
	at java.desktop/java.awt.KeyboardFocusManager.redispatchEvent(KeyboardFocusManager.java:1954)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchKeyEvent(DefaultKeyboardFocusManager.java:886)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.preDispatchKeyEvent(DefaultKeyboardFocusManager.java:1162)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.typeAheadAssertions(DefaultKeyboardFocusManager.java:1023)
	at java.desktop/java.awt.DefaultKeyboardFocusManager.dispatchEvent(DefaultKeyboardFocusManager.java:849)
	at com.intellij.ide.IdeKeyboardFocusManager.dispatchEvent$lambda$1(IdeKeyboardFocusManager.kt:48)
	at com.intellij.ide.IdeKeyboardFocusManager.dispatchEvent$lambda$2(IdeKeyboardFocusManager.kt:54)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:958)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:109)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:958)
	at com.intellij.ide.IdeKeyboardFocusManager.dispatchEvent(IdeKeyboardFocusManager.kt:54)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:4920)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2810)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4871)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:783)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:98)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:755)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:753)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:752)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:585)
	at com.intellij.ide.IdeEventQueue.dispatchKeyEvent(IdeEventQueue.kt:518)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$17(IdeEventQueue.kt:474)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction$lambda$6(AnyThreadWriteThreadingSupport.kt:274)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWithTemporaryThreadLocal(AnyThreadWriteThreadingSupport.kt:204)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:274)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runPreventiveWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:218)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:474)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$12$lambda$11$lambda$10$lambda$9(IdeEventQueue.kt:307)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:864)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$12$lambda$11$lambda$10(IdeEventQueue.kt:306)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:958)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:109)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:958)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$12(IdeEventQueue.kt:301)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:341)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:121)
	at java.desktop/java.awt.WaitDispatchSupport$2.run(WaitDispatchSupport.java:191)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:236)
	at java.desktop/java.awt.WaitDispatchSupport$4.run(WaitDispatchSupport.java:234)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:319)
	at java.desktop/java.awt.WaitDispatchSupport.enter(WaitDispatchSupport.java:234)
	at java.desktop/java.awt.Dialog.show(Dialog.java:1079)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl$MyDialog.show(DialogWrapperPeerImpl.java:871)
	at com.intellij.openapi.ui.impl.DialogWrapperPeerImpl.show(DialogWrapperPeerImpl.java:458)
	at com.intellij.openapi.ui.DialogWrapper.doShow(DialogWrapper.java:1815)
	at com.intellij.openapi.ui.DialogWrapper.show(DialogWrapper.java:1764)
	at com.bloc.intellij_generator_plugin.action.GenerateBlocAction.actionPerformed(GenerateBlocAction.kt:19)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.doPerformActionOrShowPopup(ActionUtil.kt:383)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks$lambda$7(ActionUtil.kt:353)
	at com.intellij.openapi.actionSystem.impl.ActionManagerImpl.performWithActionCallbacks(ActionManagerImpl.kt:1167)
	at com.intellij.openapi.actionSystem.ex.ActionUtil.performActionDumbAwareWithCallbacks(ActionUtil.kt:352)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.performAction$lambda$5(ActionMenuItem.kt:273)
	at com.intellij.openapi.wm.impl.FocusManagerImpl.runOnOwnContext(FocusManagerImpl.java:231)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.performAction(ActionMenuItem.kt:265)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem._init_$lambda$0(ActionMenuItem.kt:72)
	at java.desktop/javax.swing.AbstractButton.fireActionPerformed(AbstractButton.java:1972)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed$lambda$4(ActionMenuItem.kt:103)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:109)
	at com.intellij.openapi.application.TransactionGuardImpl.performUserActivity(TransactionGuardImpl.java:98)
	at com.intellij.openapi.actionSystem.impl.ActionMenuItem.fireActionPerformed(ActionMenuItem.kt:102)
	at com.intellij.ui.plaf.beg.BegMenuItemUI.doClick(BegMenuItemUI.java:521)
	at com.intellij.ui.plaf.beg.BegMenuItemUI$MyMouseInputHandler.mouseReleased(BegMenuItemUI.java:554)
	at java.desktop/java.awt.Component.processMouseEvent(Component.java:6673)
	at java.desktop/javax.swing.JComponent.processMouseEvent(JComponent.java:3394)
	at java.desktop/java.awt.Component.processEvent(Component.java:6438)
	at java.desktop/java.awt.Container.processEvent(Container.java:2266)
	at java.desktop/java.awt.Component.dispatchEventImpl(Component.java:5043)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2324)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4871)
	at java.desktop/java.awt.LightweightDispatcher.retargetMouseEvent(Container.java:4963)
	at java.desktop/java.awt.LightweightDispatcher.processMouseEvent(Container.java:4577)
	at java.desktop/java.awt.LightweightDispatcher.dispatchEvent(Container.java:4518)
	at java.desktop/java.awt.Container.dispatchEventImpl(Container.java:2310)
	at java.desktop/java.awt.Window.dispatchEventImpl(Window.java:2810)
	at java.desktop/java.awt.Component.dispatchEvent(Component.java:4871)
	at java.desktop/java.awt.EventQueue.dispatchEventImpl(EventQueue.java:783)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:728)
	at java.desktop/java.awt.EventQueue$4.run(EventQueue.java:722)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:98)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:755)
	at java.desktop/java.awt.EventQueue$5.run(EventQueue.java:753)
	at java.base/java.security.AccessController.doPrivileged(AccessController.java:400)
	at java.base/java.security.ProtectionDomain$JavaSecurityAccessImpl.doIntersectionPrivilege(ProtectionDomain.java:87)
	at java.desktop/java.awt.EventQueue.dispatchEvent(EventQueue.java:752)
	at com.intellij.ide.IdeEventQueue.defaultDispatchEvent(IdeEventQueue.kt:585)
	at com.intellij.ide.IdeEventQueue.dispatchMouseEvent(IdeEventQueue.kt:530)
	at com.intellij.ide.IdeEventQueue._dispatchEvent$lambda$16(IdeEventQueue.kt:473)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction$lambda$6(AnyThreadWriteThreadingSupport.kt:274)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWithTemporaryThreadLocal(AnyThreadWriteThreadingSupport.kt:204)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:274)
	at com.intellij.openapi.application.impl.AnyThreadWriteThreadingSupport.runPreventiveWriteIntentReadAction(AnyThreadWriteThreadingSupport.kt:218)
	at com.intellij.ide.IdeEventQueue._dispatchEvent(IdeEventQueue.kt:473)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$12$lambda$11$lambda$10$lambda$9(IdeEventQueue.kt:307)
	at com.intellij.openapi.progress.impl.CoreProgressManager.computePrioritized(CoreProgressManager.java:864)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$12$lambda$11$lambda$10(IdeEventQueue.kt:306)
	at com.intellij.ide.IdeEventQueueKt.performActivity$lambda$3(IdeEventQueue.kt:958)
	at com.intellij.openapi.application.TransactionGuardImpl.performActivity(TransactionGuardImpl.java:117)
	at com.intellij.ide.IdeEventQueueKt.performActivity(IdeEventQueue.kt:958)
	at com.intellij.ide.IdeEventQueue.dispatchEvent$lambda$12(IdeEventQueue.kt:301)
	at com.intellij.ide.IdeEventQueue.dispatchEvent(IdeEventQueue.kt:341)
	at java.desktop/java.awt.EventDispatchThread.pumpOneEventForFilters(EventDispatchThread.java:207)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForFilter(EventDispatchThread.java:128)
	at java.desktop/java.awt.EventDispatchThread.pumpEventsForHierarchy(EventDispatchThread.java:117)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:113)
	at java.desktop/java.awt.EventDispatchThread.pumpEvents(EventDispatchThread.java:105)
	at java.desktop/java.awt.EventDispatchThread.run(EventDispatchThread.java:92)
	Suppressed: com.intellij.util.ExceptionUtilRt$RethrownStack: Rethrown at
		at com.intellij.util.ExceptionUtilRt.addRethrownStackAsSuppressed(ExceptionUtilRt.java:41)
		at com.intellij.util.ExceptionUtilRt.rethrowUnchecked(ExceptionUtilRt.java:16)
		at com.intellij.util.ExceptionUtil.rethrowUnchecked(ExceptionUtil.java:131)
		at com.intellij.openapi.command.impl.CommandProcessorImpl.finishCommand(CommandProcessorImpl.java:27)
		at com.intellij.openapi.command.impl.CoreCommandProcessor.lambda$executeCommand$1(CoreCommandProcessor.java:236)
		at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$executeNonCancelableSection$3(CoreProgressManager.java:289)
		at com.intellij.openapi.progress.impl.CoreProgressManager.registerIndicatorAndRun(CoreProgressManager.java:755)
		at com.intellij.openapi.progress.impl.CoreProgressManager.computeUnderProgress(CoreProgressManager.java:711)
		at com.intellij.openapi.progress.impl.CoreProgressManager.lambda$computeInNonCancelableSection$4(CoreProgressManager.java:297)
		at com.intellij.openapi.progress.Cancellation.computeInNonCancelableSection(Cancellation.java:139)
		at com.intellij.openapi.progress.impl.CoreProgressManager.computeInNonCancelableSection(CoreProgressManager.java:297)
		at com.intellij.openapi.progress.impl.CoreProgressManager.executeNonCancelableSection(CoreProgressManager.java:288)
		at com.intellij.openapi.command.impl.CoreCommandProcessor.lambda$executeCommand$2(CoreCommandProcessor.java:235)
		... 156 more
Caused by: java.lang.ClassNotFoundException: org.apache.commons.text.StringSubstitutor PluginClassLoader(plugin=PluginDescriptor(name=Bloc, id=com.bloc.intellij_generator_plugin, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/IdeaIC2025.1/plugins/intellij_generator_plugin, version=4.1.1, package=null, isBundled=false), packagePrefix=null, state=active, parents=PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.java.terminal, descriptorPath=intellij.java.terminal.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=com.intellij.java.terminal, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.java.performancePlugin, descriptorPath=intellij.java.performancePlugin.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=com.intellij.java.performancePlugin, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.java.debugger.impl.frontend, descriptorPath=intellij.java.debugger.impl.frontend.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=com.intellij.java.debugger.impl.frontend, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.java.debugger.impl.backend, descriptorPath=intellij.java.debugger.impl.backend.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=com.intellij.java.debugger.impl.backend, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.java.compiler.charts, descriptorPath=intellij.java.compiler.charts.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=com.intellij.java.compiler.charts, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.impl, descriptorPath=intellij.platform.vcs.impl.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/lib, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.log.impl, descriptorPath=intellij.platform.vcs.log.impl.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/lib, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.vcs.dvcs.impl, descriptorPath=intellij.platform.vcs.dvcs.impl.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/lib, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.jvm.analysis.impl, descriptorPath=intellij.jvm.analysis.impl.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.java.vcs, descriptorPath=intellij.java.vcs.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.java.unscramble, descriptorPath=intellij.java.unscramble.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.java.structuralSearch, descriptorPath=intellij.java.structuralSearch.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=IDEA CORE, id=com.intellij, moduleName=intellij.platform.collaborationTools, descriptorPath=intellij.platform.collaborationTools.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/lib, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, moduleName=intellij.java.featuresTrainer, descriptorPath=intellij.java.featuresTrainer.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=Java, id=com.intellij.java, descriptorPath=plugin.xml, path=~/Applications/IntelliJ IDEA Community Edition.app/Contents/plugins/java, version=251.25410.109, package=null, isBundled=true), PluginDescriptor(name=LSP4IJ, id=com.redhat.devtools.lsp4ij, descriptorPath=plugin.xml, path=~/Library/Application Support/JetBrains/IdeaIC2025.1/plugins/lsp4ij, version=0.12.0, package=null, isBundled=false), )
	... 160 more

```

--- a/extensions/intellij/intellij_generator_plugin/build.gradle.kts
+++ b/extensions/intellij/intellij_generator_plugin/build.gradle.kts
@@ -1,7 +1,7 @@
 plugins {
     java
     id("org.jetbrains.intellij.platform") version "2.5.0"
-    kotlin("jvm") version "1.9.25"
+    kotlin("jvm") version "2.1.0"
     idea
 }
 
@@ -11,6 +11,7 @@ version = "4.1.1"
 val lsp4ijVersion: String by project
 val lsp4jVersion: String by project
 val caseFormatVersion: String by project
+val apacheCommonsTextVersion: String by project
 
 java {
     sourceCompatibility = JavaVersion.VERSION_17
@@ -53,6 +54,7 @@ dependencies {
     compileOnly("com.redhat.devtools.intellij:lsp4ij:$lsp4ijVersion")
     compileOnly("org.eclipse.lsp4j:org.eclipse.lsp4j:$lsp4jVersion")
     implementation("com.fleshgrinder.kotlin:case-format:$caseFormatVersion")
+    implementation("org.apache.commons:commons-text:$apacheCommonsTextVersion")
 }
 
 tasks {

--- a/extensions/intellij/intellij_generator_plugin/build.gradle.kts
+++ b/extensions/intellij/intellij_generator_plugin/build.gradle.kts
@@ -1,3 +1,5 @@
+import org.jetbrains.kotlin.gradle.dsl.JvmTarget
+
 plugins {
     java
     id("org.jetbrains.intellij.platform") version "2.5.0"
@@ -59,10 +61,14 @@ dependencies {
 
 tasks {
     compileKotlin {
-        kotlinOptions.jvmTarget = "17"
+        compilerOptions {
+            jvmTarget.set(JvmTarget.JVM_17)
+        }
     }
 
     compileTestKotlin {
-        kotlinOptions.jvmTarget = "17"
+        compilerOptions {
+            jvmTarget.set(JvmTarget.JVM_17)
+        }
     }
 }

--- a/extensions/intellij/intellij_generator_plugin/gradle.properties
+++ b/extensions/intellij/intellij_generator_plugin/gradle.properties
@@ -1,7 +1,7 @@
 kotlin.code.style=official
 kotlin.stdlib.default.dependency=true
-
 # Libraries versions
 lsp4jVersion=0.21.1
 lsp4ijVersion=0.12.0
 caseFormatVersion=0.2.0
+apacheCommonsTextVersion=1.13.1

